### PR TITLE
WIP change/remove subsidies

### PR DESCRIPTION
**Pull request by thebailers**

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala [hunk 1]
```diff
@@ -415,9 +415,11 @@ class SubsidyController @Inject()(
       "should-store-trader-ref" -> mandatory("should-claim-eori"),
       "claim-trader-ref" -> optional(text)
     )(OptionalTraderRef.apply)(OptionalTraderRef.unapply)
-      .transform[OptionalTraderRef](
+    .transform[OptionalTraderRef](
       optionalTraderRef => if (optionalTraderRef.setValue == "false") optionalTraderRef.copy(value = None) else optionalTraderRef,
-        identity
+      identity
+    ).verifying(
+      "error.isempty", a => a.setValue == "false" || a.value.nonEmpty
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala:422]: entry for "add-claim-trader-reference.error.isempty"  in messages.cy file is missing

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddTraderReferencePage.scala.html [hunk 2]
```diff
@@ -84,12 +85,12 @@
...
             content = Text(messages("common.yes")),
             value = Some("true"),
-            checked = form.data.get("should-store-trader-ref").contains("true"),
+            checked = form.data.get("claim-trader-ref").nonEmpty,
             conditionalHtml = Some(textHtml)
           ),
           RadioItem(
             content = Text(messages("common.no")),
-            checked = form.data.get("should-store-trader-ref").contains("false"),
+            checked = form.data.get("claim-trader-ref").isEmpty,
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddTraderReferencePage.scala.html:93]: Will this not cause "NO"to be checked by default?

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala (hunk 1 above)
Comment (thebailers) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala:422]: Added placeholder, waiting for the copy

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddTraderReferencePage.scala.html (hunk 2 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddTraderReferencePage.scala.html:93]: I test this in unit test cases and by default it gives the value checked as false. if you revert it to the previous one, t works fine

**Review by priyankadhiman3006 — APPROVED** [review]

**Review by priyankadhiman3006 — APPROVED** [review]

**Pull request merged**